Formula/cgal.rb

```diff
@@ -10,54 +10,58 @@ class Cgal < Formula
     sha256 "642c67e59a4bcfc9e22a15f2f7149045ce037c639eb9a8cf2deb3d3195e69ad0" => :mojave
     sha256 "642c67e59a4bcfc9e22a15f2f7149045ce037c639eb9a8cf2deb3d3195e69ad0" => :high_sierra
   end
-
+  
   depends_on "cmake" => [:build, :test]
   depends_on "boost"
+  depends_on "cgal"
   depends_on "eigen"
   depends_on "gmp"
   depends_on "mpfr"
+  depends_on "qt" => [:build, :test]
 
   def install
-    args = std_cmake_args + %W[
+    args = std_cmake_args + %w[
       -DCMAKE_CXX_FLAGS='-std=c++14'
-      -DWITH_Eigen3=ON
-      -DWITH_LAPACK=ON
-      -DWITH_CGAL_Qt5=OFF
-      -DWITH_CGAL_ImageIO=OFF
+      -DWITH_CGAL_Qt5=ON
+    ]
+
+    args_qt5 = %w[
+      -DCOMPONENT=CGAL_Qt5
+      -P
+      cmake_install.cmake
     ]
 
     system "cmake", ".", *args
-    system "make", "install"
+    system "cmake", *args_qt5
   end
-
   test do
-    # https://doc.cgal.org/latest/Algebraic_foundations/Algebraic_foundations_2interoperable_8cpp-example.html
```

Comment:
I'd prefer to have both test cases here. That makes it easier to see where the problem is.